cards.js

```diff
@@ -17,7 +17,27 @@ function shuffle(array){
 }
 
 function buildDeck(){
+  var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
+  var suits   = ["Clubs", "Diamonds", "Hearts", "Spades"];
+  var deck = []
+  for(v in values){
+    for(s in suits){
+      var new_card = {card_value: (values[v]), suitcard_value: (suits[s])}
+      deck.push(new_card);
+    }
+  }
+  var shuffled_deck = shuffle(deck)
+  console.log(shuffled_deck);
+}
 
-// Put some code in here!
+var deck = buildDeck()
 
+console.log(deck)
+
+function high_card(deck){
+    var player1 = deck.pop()
+    var player2 = deck.pop()
+  console.log(player1, player2)
```

Comment:
You're off to a good start! The next thing I would do here is compare the values of `player1.suitcard_value` and `player2.suitcard_value`